### PR TITLE
Pull request second attempt

### PR DESCRIPTION
**Pull request by lacurnow**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -1,8 +1,48 @@
...
+  end
+
+  get '/play' do
+    @player_name = $player.name
+    erb :play
+  end
+
+  get '/choice' do
+    erb :choice
+  end
```
Comment (estoltzy) [inline app.rb:32]: Great idea to use radio buttons! I hadn't thought of that

File: lib/player.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Player
+  attr_reader :name 
+  attr_accessor :choice
+
+  def initialize(name)
```
Comment (estoltzy) [inline lib/player.rb:5]: When you initialize two global variables here, do you need two parameters? How does the player register their choice? (a genuine question, not saying your code is wrong!)

File: lib/game.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      else
+        winner = "Draw"
+      end
+    elsif player_choice == "Paper"
+      if cpu_weapon == "Scissors"
+        winner = cpu
+      elsif cpu_weapon == "Rock"
+        winner = player
+      else
+        winner == "Draw"
```
Comment (estoltzy) [inline lib/game.rb:33]: You have == rather than = here

File: spec/features/choice_selector_spec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  scenario 'choices are displayed' do
+    enter_name_and_play
+    click_button 'Start'
+    expect(page).to have_content "Rock, Paper, or Scissors?\nMake your choice."
+  end
+
+  scenario 'player can select rock' do
+    enter_name_and_play
+    click_button 'Start'
+    choose 'Rock', allow_label_click: true
```
Comment (estoltzy) [inline spec/features/choice_selector_spec.rb:11]: I hadn't seen this allow_label_click: true before. Such a useful way of testing a specific outcome.

File: views/outcome.erb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<h1><%= @player_name%> picked <%=@player_choice%>!</h1>
+<h1>CPU picked <%= $computer_choice%>!</h1>
+<h1><%=@winner%> is the winner!</h1>
```
Comment (estoltzy) [inline views/outcome.erb:3]: Where is this @winner variable? I might be missing it somewhere, but in your app.rb you define the winner as $winner. Is that part of the issue of it not printing the name?

File: lib/player.rb (hunk 2 above)
Comment (alicelieutier) [reply lib/player.rb:5]: Agreed - where doe the choice come from? If the choice is not known at initialisation, then you could do `@choice = nil` instead, to show it's nil by default.

**Review by alicelieutier — COMMENTED** [review]
Well organised. I think you have definitely reached the main goals of the week, but you seem to have a few bugs :)
Worth practicing debugging with lots of getting visibility (`p` and `save_and_open_page` will help).

**Pull request closed without merging**